extensions/blocks/send-a-message/whatsapp-button/index.js

```diff
@@ -6,6 +6,7 @@ import { __, _x } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import { getIconColor } from '../../shared/block-icons';
```

Comment:
Since we're one level downer, we're going to have to update that path:

```suggestion
import { getIconColor } from '../../../shared/block-icons';
```